pages/blog/announcing-graphql-yoga-2.mdx

```diff
@@ -0,0 +1,293 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'Announcing GraphQL Yoga 2.0!',
+  tags: ['server', 'envelop', 'node', 'graphql'],
+  authors: ['charlypoly', 'arda', 'laurin', 'saihaj'],
+  date: '2022-03-29',
+  description:
+    'Fully-featured GraphQL Server with focus on easy setup, performance and great developer experience',
+  image: '/blog-assets/announcing-graphql-yoga-2/cover.png',
+  thumbnail: '/blog-assets/announcing-graphql-yoga-2/thumbnail.png'
+}
+
+export default withArticle({ ...meta })
+
+Today we are incredibly excited to share with you the new GraphQL Yoga! This release was made possible with your contributions, issues, and feedback.
+
+[The Guild](https://www.the-guild.dev/about-us) took over the development of GraphQL Yoga from [Prisma](https://www.prisma.io/) in early 2021, and with the growing community of tools in the GraphQL space, most recently [Envelop](https://envelop.dev/), we were able to rewrite GraphQL Yoga 2.0 from scratch with easy setup, performance, and developer experience at the core.
+
+GraphQL Yoga continues to boast a "convention over configuration" approach and the freedom to use your favorite libraries, from HTTP to schema-building.
+
+No longer are you required to install dozens of dependencies to get features such as Subscriptions, File Uploads, CORS, Error Masking, and more.
```

Comment:
I wouldn't consider these features proper nouns and therefore would rather lowercase them.

```suggestion
No longer are you required to install dozens of dependencies to get features such as subscriptions, file uploads, CORS, error masking, and more.
```